client/app/scripts/liveblog-edit/module.js

```diff
@@ -246,9 +247,7 @@ define([
                 }
                 vm.tab = tab;
             },
-            // take the public url (from s3) or the local address
-            // FIXME: The local address shouldn't be given on production mode
-            iframe_url: blog.public_url || config.server.url.replace('/api', '/embed/' + blog._id),
```

Comment:
If you remove this line, I think that `config` isn't needed any more here, you can remove it from the dependencies